OWML.ModHelper.Menus/ModComboInput.cs

```diff
@@ -0,0 +1,139 @@
+﻿using OWML.Common.Menus;
+using OWML.ModHelper.Events;
+using OWML.Common;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModComboInput : ModInput<string>, IModComboInput
+    {
+
+        private string _value;
+        private Vector3 scale;
+        private UIStyleManager styleManager;
+        private HorizontalLayoutGroup _layoutGroup;
+        public IModLayoutButton Button { get; }
+        protected readonly IModInputMenu InputMenu;
+        protected readonly TwoButtonToggleElement ToggleElement;
+
+        public ModComboInput(TwoButtonToggleElement element, IModMenu menu, IModInputMenu inputMenu) : base(element, menu)
+        {
+            ToggleElement = element;
+            InputMenu = inputMenu;
+            scale = element.GetValue<Button>("_buttonTrue").transform.localScale;
+            Button = new ModLayoutButton(element.GetValue<Button>("_buttonTrue"), menu);
+            Button.OnClick += Open;
+            var noButton = ToggleElement.GetValue<Button>("_buttonFalse");
+            noButton.transform.parent.gameObject.SetActive(false);
+            _layoutGroup = Button.LayoutGroup;
+            ((RectTransform)_layoutGroup.transform).sizeDelta = new Vector2(((RectTransform)Button.Button.transform.parent).sizeDelta.x * 2, ((RectTransform)Button.Button.transform.parent).sizeDelta.y);
+            styleManager = MonoBehaviour.FindObjectOfType<UIStyleManager>();
+
+            var layoutGroup = Button.Button.transform.parent.parent.GetComponent<HorizontalLayoutGroup>();
+            layoutGroup.childControlWidth = true;
+            layoutGroup.childForceExpandWidth = true;
+            Button.Button.transform.parent.GetComponent<LayoutElement>().preferredWidth = 100;
+        }
+
+        private void UpdateLayout(string currentCombination)
+        {
+            int cnt = _layoutGroup.transform.childCount;
+            for (int i = cnt - 1; i >= 0; i--)
```

Comment:
Use curly brackets for if, for, foreach, etc.

Reply:
probably done (hope I didn't miss any this time)